zenoh/src/api/builders/publisher.rs

```diff
@@ -294,6 +309,16 @@ impl<'a, 'b> PublisherBuilder<'a, 'b> {
         self
     }
 
+    /// Change the `reliability`` to apply when routing the data.
```

Comment:
```suggestion
    /// Change the `reliability`` to apply when routing the data.
    /// NOTE: Currently `reliability` does not trigger any data retransmission on the wire. 
    ///             It is rather used as a marker on the wire and it may be used to select the best link available (e.g. TCP for reliable data and UDP for best effort data).
```